eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -34,7 +34,7 @@ parameters:
 # Skip running the SDK content smoke-tests
 - name: excludeSdkContentTests
```

Comment:
I know @premun said this could be left in place but IMO it should be cleaned up.  It just adds complexity.  We should be simplifying the pipeline where possible and not maintaining unused code paths.

Reply:
Makes sense, let's kill all the code around this then!